### PR TITLE
Add crictl config.

### PR DESCRIPTION
**Pull request by Random-Liu**
Add a default `crictl` confg. @yanxuean 

Also remove a redundant log.
Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: pkg/server/container_attach.go [hunk 1]
```diff
@@ -29,13 +29,6 @@ import (
 
 // Attach prepares a streaming endpoint to attach to a running container, and returns the address.
 func (c *criContainerdService) Attach(ctx context.Context, r *runtime.AttachRequest) (retRes *runtime.AttachResponse, retErr error) {
```
Comment (yanxuean) [inline pkg/server/container_attach.go]: nit:  The name of argument may be removed.  e.g. retRes, retErr

Comment (Random-Liu) [reply pkg/server/container_attach.go]: Will do

Comment (Random-Liu) [reply pkg/server/container_attach.go]: Done.

**Pull request merged**